logging/context.go

```diff
@@ -0,0 +1,112 @@
+// MIT License. Copyright (c) 2020 CQFN
+// https://github.com/cqfn/degitx/blob/master/LICENSE
+
+package logging
+
+import (
+	"errors"
+
+	"cqfn.org/degitx/locators"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type LogConfig struct {
+	Outputs   []Output `yaml:"outputs"`
+	ErrorsOut []string `yaml:"errors"`
+}
+
+type Output struct {
+	UniqName string   `yaml:"u_name"`
```

Comment:
@stepanvalyavskiy do we really need this name field? Maybe we can safely remove it?

Reply:
We have a logger with multicores, so it's one logger that has different log level for several output groups.
For ex `DEBUG` for `stdout` & `file path 2` 
and `WARN` for `file path 2` & `some url`
If we want to change a log level only for one output group, it's impossible to `log.SetLevel($LVL)`, we need to ident output grpoup at first. Otherwise all output groups will became same log level after calling `log.SetLevel($LVL)`
To change log level only for one group I use  `log.SetLevel($LVL, $OUTPUT_UID)`

TLDR: If we remove this field, we lose an opportunity to change log level only for some output groups.

Reply:
@stepanvalyavskiy what is the output group? I think we can initialize log from config only once, so we don't need `SetLogLevel()` method at all: we can just parse the config, and construct logger for each item from logging configuration list

Reply:
> @stepanvalyavskiy what is the output group? I think we can initialize log from config only once, so we don't need `SetLogLevel()` method at all: we can just parse the config, and construct logger for each item from logging configuration list

then we can remove SetLevel and u_name as well.